tests/integration-tests/tests/storage/test_efs/test_multiple_efs/pcluster.config.yaml

```diff
@@ -45,4 +45,6 @@ SharedStorage:
     StorageType: Efs
     EfsSettings:
       FileSystemId: {{ efs_fs_id }}
+      IamAuthorization: {{ iam_authorizations[loop.index-1] }}
+      EncryptionInTransit: {{ encryption_in_transits[loop.index-1] }}
```

Comment:
This should only be done if scheduler is not AWS Batch

Reply:
Why would this not be done if we are setting the boolean array to all false as specified above? https://github.com/aws/aws-parallelcluster/pull/4668#discussion_r1048832724

Reply:
Is it due to the omission here https://github.com/aws/aws-parallelcluster/pull/4668#discussion_r1048832038?

Reply:
On further inspection, this change is not necessary if the boolean arrays are set to all false and passed through pcluster_config_reader(...)